src/main/java/com/metamx/metrics/SigarUtil.java

```diff
@@ -58,9 +58,20 @@
     }
   }
 
+  /**
+   * SigarHolder class is initialized after SigarUtil, that guarantees that SIGAR = new Sigar() is executed after static
+   * block (which loads the library) of SigarUtil is executed. This is anyway guaranteed by JLS if SIGAR static field
+   * goes below the static block in textual order, but fragile e. g. if someone applies automatic reformatting and the
+   * static field is moved above the static block.
+   */
+  private static class SigarHolder
+  {
+    private static final Sigar SIGAR = new Sigar();
+  }
+
   public static Sigar getSigar()
   {
-    return new Sigar();
+    return SigarHolder.SIGAR;
```

Comment:
I'm not an expert in internals of Sigar, but I wondering can it have some side-effects for long running applications? Also it might affect somehow metrics which is being gathered and Sigar itself might be not thread-safe. I would suggest investigating more all these questions and separate this PR with changes in `SigarUtil`.